poller/poller-core/index.js

```diff
@@ -217,9 +217,12 @@ function postPubSubMessage(spanner, metrics) {
 }
 
 function callScalerHTTP(spanner, metrics) {
+  spanner.scalerURL ||= 'http://scaler';
+  const url = new URL('/metrics', spanner.scalerURL);
+  
```

Comment:
Nit: possible to remove the whitespace on this line?

Reply:
done!